### DISCUSSION TITLE
Hand tool shortcut ?

### DISCUSSION THREAD

**Discussion by Shermaaann** [xournalpp/xournalpp#4358] [Q&A] [answered]
![xournal hand](https://user-images.githubusercontent.com/116521465/197417121-d5c62110-a5b7-4a75-9ac1-31d1aad9949a.png)
Hi, I was wondering if there was a keyboard shortcut for the hand/drag/pan tool. I can't find it in the tool menu and I want to assign it to a button on my graphics tablet.

**Comment by rolandlo** [comment, marked as answer]
Yes, it is `<Shift><Ctrl>A` as you can see from menu `Tools`. By the way, if there were no keyboard shortcut, you could easily write a [plugin](https://xournalpp.github.io/guide/plugins/plugins/) that creates one.

**Reply by Shermaaann** [reply]
Thank you, it is exactly what I was searching for !
